docroot/modules/custom/nvli_custom_search/src/Plugin/Block/NewSearchBlock.php

```diff
@@ -25,7 +25,7 @@ class NewSearchBlock extends BlockBase {
    * {@inheritdoc}
    */
   protected function blockAccess(AccountInterface $account) {
```

Comment:
@piyuesh23 : This block will render the search form.
This file I already pushed but now I change the permission.